src/manager/game.hpp

```diff
@@ -15,10 +15,19 @@ namespace manager {
         Logger logger_;
 
       public:
+        /**
+         * @brief Inizializza il gioco
```

Comment:
boh, secondo me davvero i commenti servono a spiegare il codice che non è autoesplicativo, in questo caso lo è